### PR TITLE
Directory organization

### PR DESCRIPTION
**Pull request by AlFasGD**
P.S. Should `GDAPI.Enumerations` also have a `General` directory? It violates the general pattern where a folder may only contain either folders or files, not both.

### CODE REVIEW

File: GDAPI/GDAPI/Utilities/Objects/GeometryDash/LevelObjects/SpecialObjects/Pads/Pad.cs [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-
-namespace GDAPI.Utilities.Objects.GeometryDash.LevelObjects.SpecialObjects.Pads
-{
-    /// <summary>Represents a pad.</summary>
-    public abstract class Pad : OrbPad
-    {
-        /// <summary>Initializes a new instance of the <seealso cref="Pad"/> class.</summary>
-        public Pad() : base() { }
-    }
-}
```
Comment (Altenhh) [inline GDAPI/GDAPI/Utilities/Objects/GeometryDash/LevelObjects/SpecialObjects/Pads/Pad.cs]: Why was this deleted?

Comment (AlFasGD) [reply GDAPI/GDAPI/Utilities/Objects/GeometryDash/LevelObjects/SpecialObjects/Pads/Pad.cs]: Git apparently preferred showing a deletion and an addition for this and an interface. Nothing was lost, otherwise the project wouldn't be built.

**Comment by Altenhh** [comment]
i dont really care what you do with the organization, just make sure to place classes on where they should be expected to be in
